mobile/src/modules/auth/components/AccountListItem.tsx

```diff
@@ -120,7 +124,11 @@ export class AccountListItem extends React.PureComponent<Props> {
             {priceApi && priceApi.priceInfo && (
               <View style={styles.row}>
                 <Text size={FontSizes.SMALL} textAlign={TextAlign.RIGHT} color={Colors.WHITE}>
-                  {i18n.t(core.currency.BTC.value, { value: balanceBTC })}
+                  {/* todo: figure out why this doesnt work */}
+                  {priceApi.selectedCurrency === 'USD' ?
+                    // i18n.t(core.currency.USD.value, { value: balanceUSD.toFixed(2) }) :
+                    i18n.t(core.currency.BTC.value, { value: amountToString(balanceBTC) }) :
+                    i18n.t(core.currency.BTC.value, { value: amountToString(balanceBTC) })}
```

Comment:
This is where I struggled to get the `props` to trigger a `render` whenever `priceApi` changed. For some reason, this component doesnt update when those props change.

Reply:
change `React.PureComponent` to `React.Component`, it's a common issue with `FlatList` items